code/modules/atmospherics/auxgm/gas_types.dm

```diff
@@ -127,3 +127,13 @@
 	name = "Miasma"
 	gas_overlay = "miasma"
 	moles_visible = MOLES_GAS_VISIBLE * 60
+
+//NUCLEIUM Waste Gas from RBMK Nuclear Reactor	//Monkestation Edit
+/datum/gas/nucleium
+	id = "nucleium"
+	specific_heat = 450 //jesus god why
```

Comment:
I can immediately see this being a problem.
This would make for a better thermal battery for TEGs and even cooling loops than plasma itself.

I'd suggest closer to 180 unless this absolutely needs to be higher.

Reply:
It should be safe to lower it, it is only and output gas from the reactor